yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -165,13 +165,15 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 
 	var obfuscatedIssues []callhome.AssessmentIssueCallhome
 	for _, issue := range assessmentReport.Issues {
+		obfusactedIssueDetails := callhome.ObfuscatedIssueDetails(issue.Details)
 		obfuscatedIssue := callhome.AssessmentIssueCallhome{
 			Category:            issue.Category,
 			CategoryDescription: issue.CategoryDescription,
 			Type:                issue.Type,
 			Name:                issue.Name,
 			Impact:              issue.Impact,
 			ObjectType:          issue.ObjectType,
+			Details:             obfusactedIssueDetails,
```

Comment:
create a constructor for AssessmentIssueCallhome and move the obfuscation logic there